### ISSUE TITLE
snowflake/llamaguard gives ReplicateError: ReplicateError Details: status: 404 detail: Not found.

### ISSUE THREAD

**Issue by olsonperrensen** [streamlit/snowflake-arctic-st-demo#30]
`File "c:\Users\X\source\polars-streamlit\.conda\lib\site-packages\streamlit\runtime\scriptrunner\script_runner.py", line 600, in _run_script
    exec(code, module.__dict__)
File "C:\Users\X\source\polars-streamlit\pages\openapi.py", line 158, in <module>
    main()
File "C:\Users\X\source\polars-streamlit\pages\openapi.py", line 13, in main
    init_chat_history()
File "C:\Users\X\source\polars-streamlit\pages\openapi.py", line 48, in init_chat_history
    check_safety()
File "C:\Users\X\source\polars-streamlit\pages\openapi.py", line 75, in check_safety
    deployment = get_llamaguard_deployment()
File "c:\Users\X\source\polars-streamlit\.conda\lib\site-packages\streamlit\runtime\caching\cache_utils.py", line 165, in wrapper
    return cached_func(*args, **kwargs)
File "c:\Users\X\source\polars-streamlit\.conda\lib\site-packages\streamlit\runtime\caching\cache_utils.py", line 196, in __call__
    return self._get_or_create_cached_value(args, kwargs)
File "c:\Users\X\source\polars-streamlit\.conda\lib\site-packages\streamlit\runtime\caching\cache_utils.py", line 221, in _get_or_create_cached_value
    return self._handle_cache_miss(cache, value_key, func_args, func_kwargs)
File "c:\Users\X\source\polars-streamlit\.conda\lib\site-packages\streamlit\runtime\caching\cache_utils.py", line 277, in _handle_cache_miss
    computed_value = self._info.func(*func_args, **func_kwargs)
File "C:\Users\X\source\polars-streamlit\pages\openapi.py", line 68, in get_llamaguard_deployment
    return replicate.deployments.get("snowflake/llamaguard")
File "c:\Users\X\source\polars-streamlit\.conda\lib\site-packages\replicate\deployment.py", line 195, in get
    resp = self._client._request(
File "c:\Users\X\source\polars-streamlit\.conda\lib\site-packages\replicate\client.py", line 87, in _request
    _raise_for_status(resp)
File "c:\Users\X\source\polars-streamlit\.conda\lib\site-packages\replicate\client.py", line 367, in _raise_for_status
    raise ReplicateError.from_response(resp)`

**Comment by sfc-gh-cfrasca** [comment]
Hey @olsonperrensen, I haven't been able to reproduce this error – are you running the app as-is locally?

**Comment by olsonperrensen** [comment]
Yes. Solved it using commit 57563341e549e7fef13df75679015adf6bd562d2 on HuggingFace.

**Comment by jason-sypniewski** [comment]
I have the same issue using replicate, same 404 error.  Seems it cannot find the snowflake/llamaguard deployment.  I am able to run the simple version of the app with no issues using my REPLICATE_API_TOKEN.

**Comment by xnileshtiwari** [comment]
"I have the same issue using replicate, same 404 error. Seems it cannot find the snowflake/llamaguard deployment. I am able to run the simple version of the app with no issues using my REPLICATE_API_TOKEN." Same here how to fix this?

**Comment by Klimmy** [comment]
I have the same problem.
@olsonperrensen, could you share the link to the commit that helped you solve the issue?

**Comment by olsonperrensen** [comment]
Hi @Klimmy sure [/spaces/Snowflake/snowflake-arctic-st-demo/tree/57563341e549e7fef13df75679015adf6bd562d2](https://huggingface.co/spaces/Snowflake/snowflake-arctic-st-demo/tree/57563341e549e7fef13df75679015adf6bd562d2) last modified by [carolinefrascasnowflake](https://huggingface.co/carolinefrascasnowflake)
